crates/trie/trie/src/trie_cursor/in_memory.rs

```diff
@@ -78,7 +78,7 @@ impl<'a, C: TrieCursor> InMemoryAccountTrieCursor<'a, C> {
         exact: bool,
```

Comment:
unclear what this is supposed to represent @rkrasiuk

Reply:
whether the exact key is sought or the one that's equal or greater to the sought key